docs/backup_and_restore.rst

```diff
@@ -123,67 +105,216 @@ Restoring
 
-Migrating
----------
+#. **Preserve important credentials**: Before re-installing SecureDrop, you 
+   will need to preserve some important credential files from the old 
+   installation on the *Admin Workstation*. They should be temporarily saved
+   somewhere in the ``~/Persistent/`` directory. 
+
+   .. code:: sh
+
+       mkdir ~/Persistent/sd-restore-creds
+
+   Copy in the following files from ``~/Persistent/securedrop/install_files/ansible-base``:
+
+   **Instances using v2 onion services:**
+   
+   - ``app-source-ths``
+   - ``app-journalist-aths``
+   - ``app-ssh-aths``
+
+
+   **Instances using v3 onion services:**
+
+   - ``app-sourcev3-ths``
+   - ``app-ssh.auth_private``
+   - ``app-journalist.auth_private``
+   -  ``tor_v3_keys.json``
+
+
+   Once you have preserved those files, you may either delete or rename the 
+   SecureDrop project root directory to something like ``securedrop.old`` and
+   delete it when the migration is complete. 
+
+#. **Prepare the new servers.** Re-clone the SecureDrop repository, and 
+   :doc:`reinstall SecureDrop from scratch <install>`. 
+
+   During configuration (``./securedrop-admin sdconfig``), set up the
+   new SecureDrop installation with the same onion 
+   services version(s) as your previous server configuration. Proceed through 
+   the installation, finishing with ``./securedrop-admin tailsconfig.``
+
+   If SSH-over-Tor is configured, run ``ssh app`` and ``ssh mon`` to add the
+   new (temporary) onion URLs to your ``known_hosts`` file. 
+
+#. **Restore the backup**: Copy the backup archive into the 
+   ``install_files/ansible-base`` directory and run
+
+   .. code:: sh
+            
+       ./securedrop-admin restore sd-backup-<your_backup_file>.tar.gz
+    
+   The restore task will proceed for some time, and then will fail with the
+   message ``ssh_exchange_identification: Connection closed by remote host``   
+   during the ``Wait for Tor to reload`` task. This is expected; during the 
+   restoration process, the Application Server's onion URL changed, causing it
+   to be unreachable.
+
+   Reboot the *Application Server*, or log in via the console and issue the
+   command ``sudo service tor reload`` to restart the Tor service.
+
+#. **Fix Tor credentials**: If you are using 
+   SSH-over-Tor and are re-using the *Admin Workstation* 
+   from your previous installation, you will need to remove the ``ssh`` 
+   alias created during re-installation, or you will see a host key 
+   verification failure warning in the next section.  
+
+   **Instances using v2 onion services:**
+
+   Inspect your ``app-ssh-aths`` file: 
+
+   .. code:: sh
+
+      cat install_files/ansible-base/app-ssh-aths
+
+   This file is in the format ``HidServAuth app_ssh_url Secret # client: admin``.
+   Type the following command, replacing ``app_ssh_url`` with the .onion URL 
+   found in the file above:
+
+   .. code:: sh
+
+      ssh-keygen -f "/home/amnesia/.ssh/known_hosts" -R "app_ssh_url"
```

Comment:
Because this command is essentially identical for both v2 and v3, I would suggest consolidating this a bit further: 1) Obtain the SSH service .onion address for the Application Server (instructions vary between v2 and v3); 2) Remove the old onion service form the list of known hosts (instructions are identical once you have the .onion address).

Regarding the sample command, it's not obvious to me at a glance that `app_ssh_url` is meant to be replaced by the user. Further up we use `<your_backup_file>` to indicate that the filename is to be substituted by the user, which has the problem of using reserved shell symbols `<` and `>` but is a pretty common pragma regardless. I'm open to other ideas, but I do think we want it to stand out a bit more.

Reply:
Updated in 151175d - let me know if this wording works for you.